jbt-it/neomdb
# #308 event details page

How to test:
- go to Veranstaltungen and click on an event to get to the event detail page or go to /veranstaltungen/8
- DO NOT TEST THE OVERVIEW PAGE THIS IS ANOTHER PR

without permission:
- check if everything is displayed correctly on desktop and mobile

with permission:
- check if the icons to delete and edit are shown
- edit event details and check if error messages are shown if you leave important information out e.g. the title
- check if the remove buttons for the participants is displayed
- remove a participant
- add a participant with the respective button

closes #346 
closes #314
